src/components/SearchForm/SearchForm.js

```diff
@@ -1,11 +1,40 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './SearchForm.css';
 
-function SearchForm() {
+function SearchForm({ getSearchResults }) {
+  const [inputValue, setInputValue] = useState('');
+  const [placeHolder, setPlaceHolder] = useState('Enter topic');
+
+  function handleInputChange(e) {
+    setInputValue(e.target.value);
+  }
+
+  function resetPlaceHolder() {
+    setPlaceHolder('Enter topic');
+  }
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    if (!inputValue) setPlaceHolder('Please enter a keyword');
+    else {
+      getSearchResults(inputValue);
+    }
+  }
+
   return (
     <form className="search-form">
-      <input className="search-form__input" placeholder="Enter topic" />
-      <button className="search-form__button">Search</button>
+      <input
+        className={`searchForm__input ${
+          placeHolder === 'Please enter a keyword' && 'searchForm__input-empty'
+        }`}
+        placeholder={placeHolder}
+        onChange={handleInputChange}
+        value={inputValue}
+        onFocus={resetPlaceHolder}
+      />
+      <button type="submit" className="search-form__button" onClick={handleSubmit}>
```

Comment:
` onClick={handleSubmit}`

please, delete it from the button

Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses